certified-connectors/absentify/apiDefinition.swagger.json

```diff
@@ -0,0 +1,492 @@
+                    "email": {
+                      "type": "string",
+                      "description": "email"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "name"
+                    },
+                    "id": {
+                      "type": "string",
+                      "description": "id"
+                    },
+                    "custom_id": {
+                      "type": "string",
+                      "description": "custom_id"
+                    }
+                  },
+                  "description": "approver_member"
+                },
+                "requester_member": {
+                  "type": "object",
+                  "properties": {
+                    "email": {
+                      "type": "string",
+                      "description": "email"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "name"
+                    },
+                    "id": {
+                      "type": "string",
+                      "description": "id"
+                    },
+                    "custom_id": {
+                      "type": "string",
+                      "description": "custom_id"
+                    }
+                  },
+                  "description": "requester_member"
+                },
+                "canceld_by_member": {
+                  "type": "object",
+                  "properties": {
+                    "email": {
+                      "type": "string",
+                      "description": "email"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "name"
+                    },
+                    "id": {
+                      "type": "string",
+                      "description": "id"
+                    },
+                    "custom_id": {
+                      "type": "string",
+                      "description": "custom_id"
+                    }
+                  },
+                  "description": "canceld_by_member"
+                },
+                "declined_by_member": {
+                  "type": "object",
+                  "properties": {
+                    "email": {
+                      "type": "string",
+                      "description": "email"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "name"
+                    },
+                    "id": {
+                      "type": "string",
+                      "description": "id"
+                    },
+                    "custom_id": {
+                      "type": "string",
+                      "description": "custom_id"
+                    }
+                  },
+                  "description": "declined_by_member"

```

Comment:
@marc-on-github All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
@sriyen-msft thanks for that feedback. Its fixed